src/Pagination/index.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Icon, Theme } from '..';
+
+import {
+  Container,
+  PaginationContainer,
+  PaginationItem,
+  PaginationGroup,
+  ArrowContainer,
+  BreakItem,
+} from './elements';
+
+/**
+ * Pagination
+ *
+ * This component is in charge of displaying
+ * a Pagination
+ *
+ * @param {number} pageCount - total number of pages in the pagination.
+ * @param {number} pageRangeDisplayed - displayed page range.
+ * @param {number} marginPageDisplayed - page displayed on the pagination marges.
```

Comment:
`displayedBoundNumber`? And the doc may be clearer with something like `number of pages to display at the beginning and end of the pagination`